.github/ISSUE_TEMPLATE/resolver-failure.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Pip resolver failure
+about: Use this issue type to report when the pip dependency resolver fails
+labels: ["K: UX", "K: crash"]
```

Comment:
Should this add `C: dependency resolution` and/or `C: new resolver implementation`?